_includes/scripts.html

```diff
@@ -1,19 +1,7 @@
-<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.0.3/jquery.min.js"></script>
-<script src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.10.3/jquery-ui.min.js"></script>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
```

Comment:
The latest version of jquery is 3.7.0 - see https://jquery.com/download/, this version is what is located at https://github.com/PCMDI/assets/tree/main/jquery